src/styles/blocks/card.scss

```diff
@@ -0,0 +1,83 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  max-width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background: $white-color;
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary-color;
+    margin: 0 0 16px;
+    padding: 0;
+  }
+
+  &__review-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+  }
+
+  &__review-label {
+    color: $main-color;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    margin: 24px 0 16px;
+  }
+
+  &__price-label {
+    color: $secondary-color;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: $font-weight-bold;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    color: $white-color;
+    background-color: $accent-color;
+    border-radius: 5px;
+    border: 1px solid $accent-color;
+
+    font-size: 14px;
+    font-weight: $font-weight-bold;
+    line-height: 16px;
+    text-transform: uppercase;
+
+    &:hover {
+      background: $white-color;
```

Comment:
If you only need a color - use the exact property - `background-color`.
This will make your code more understandable